src/Resources/assets/styles/_tailwind.scss

```diff
@@ -129,6 +129,10 @@
     min-height: 2.2rem;
 }
 
+.whatwedo_core-checkbox {
+    @apply text-primary-500 focus:ring-1 focus:ring-primary-500 rounded cursor-pointer;
```

Comment:
Nit: Checkboxes and other form elements usually do not have a pointer cursor by default (not even natively in the operating system). But I also do not have any information where this requirement is coming from.

Reply:
@renestalder Interesting insights. I didn't know that.

I took this attribute from another field and wasn't thinking about the. general behaviour of checkbox.
Would you recommend keeping it there or removing it?